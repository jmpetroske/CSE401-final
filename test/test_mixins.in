$yellow = #ff0000;
$color = brown;

@mixin alert($color, $multiple) {
    background-color: $color;
    font-size: 10px * $multiple;
}

.selector {
    @use alert($yellow, 2);
}
