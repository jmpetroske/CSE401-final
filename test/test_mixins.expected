.selector {
background-color: #ff0000;
font-size: 20px;
}
